examples/hello-world/client/components/App.js

```diff
@@ -1,8 +1,27 @@
 import React from 'react'
+import { Helmet } from 'react-helmet'
+import Header from './Header.js'
 
 const RootComponent = () => (
   <div>
-    <h1>Hello, world!</h1>
+    <Helmet>
+      <link
+        rel="stylesheet"
+        href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
+        integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u"
+        crossorigin="anonymous"
+      />
+      <link
+        rel="stylesheet"
+        href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css"
```

Comment:
Looks like redundant link.

Reply:
Yes, you're right, I'll delete it. Thank you!